universal-login-relayer/lib/routes/wallet.ts

```diff
@@ -27,10 +27,26 @@ messageHandler : MessageHandler) => async (req : Request, res : Response, next :
   }
 };
 
-export const getStatus = (transactionContractService: MessageHandler) => async (req: Request, res: Response, next: NextFunction) => {
+export const getStatus = (messageHandler: MessageHandler) => async (req: Request, res: Response, next: NextFunction) => {
   try {
     const {hash} = req.params;
-    const status = await transactionContractService.getStatus(hash);
+    const status = await messageHandler.getStatus(hash);
+    res.status(200)
+      .type('json')
+      .send(JSON.stringify(status));
+  } catch (err) {
+    next(err);
+  }
+};
+
+export const getStatusById = (messageHandler: MessageHandler) => async (req: Request, res: Response, next: NextFunction) => {
+  try {
+    const {id} = req.params;
+    const messageEntity = await messageHandler.getQueueStatus(id);
+    const status = messageEntity && {
+      transactionHash: messageEntity.hash,
+      error: messageEntity.error
+    };
```

Comment:
Why not simply:
```ts
res.status(200)
    .type('json')
    .send(messageEntity);
```

Reply:
Needed only two fields and wanted to change hash for transactionHash until we change two stores to one